WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/AddProductName/AddProductNameWithAIViewModel.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+
+final class AddProductNameWithAIViewModel: ObservableObject {
```

Comment:
Nit: comment for new type.

```suggestion
/// View model for `AddProductNameWithAIView`.
///
final class AddProductNameWithAIViewModel: ObservableObject {
```

Reply:
Added in https://github.com/woocommerce/woocommerce-ios/pull/10723/commits/3f323b87dbc2c2ed5e5a0682c249435a1483a8a7